src/Sulu/Bundle/AdminBundle/Admin/View/FormOverlayListViewBuilder.php

```diff
@@ -51,13 +51,6 @@ public function setFormKey(string $formKey): FormOverlayListViewBuilderInterface
         return $this;
     }
 
-    public function setRequestParameters(array $requestParameters): FormOverlayListViewBuilderInterface
```

Comment:
That's a pretty hard BC break IMO. I would keep the `setRequestParameters` function, and add it to the ViewBuilders where it is missing..

Reply:
However, if we want to remove this function we could deprecate it. But I don't really see a reason to do so, because this probably won't make any huge problems in the future.

Reply:
the function was in no interface - so i would remove it because we never have setters (always add methods)

Reply:
You even had to change some code in our codebase, because of this change... And people had no other choice than using this function if they wanted to add some hardcoded values, so I think there is quite a high chance that people have already used.

Of course you could argue, that we only had setters until now, but IMO that still does not justify the total removal of this method. If we say we don't want to have that method in the future, I would just deprecate it, and mention the `addRequestParameters` method in the deprecation message.

But I think I would go even further, and just keep it as it is, and maybe even add it in the interface, because it is a very simple function, that probably won't add a lot of maintenance burden.